Cue/app/setup.js

```diff
@@ -29,6 +30,7 @@ function setup(): ReactClass<{}> {
       if (this.state.isLoading) {
         return null;
       }
+      CueApi.setAuthHeader(this.state.store.getState().user.userId, this.state.store.getState().user.accessToken);
```

Comment:
Actually this is ok too, since we'll only ever reach this line if `isLoading` is false, meaning the callback has been called.